bridges/snowbridge/pallets/system-frontend/src/lib.rs

```diff
@@ -271,6 +397,12 @@ pub mod pallet {
 				.reanchored(&T::BridgeHubLocation::get(), &T::UniversalLocation::get())
 				.map_err(|_| Error::<T>::LocationConversionFailed)
 		}
+
+		pub fn account_to_location(account: AccountIdOf<T>) -> Result<Location, Error<T>> {
```

Comment:
So I believe the implementation of `T::AccountId` supports `impl Into<Location>`, so you should rather use that.

To get that to work, you'll need to appropriate bounds like:

```rust
impl<T: Config> Pallet<T> where T::AccountId: Into<Location> {
```

But I can't remember properly. You may need to ask Parity.

Reply:
it could also `where Location: From<<T as frame_system::Config>::AccountId> `. Some previous discussion in https://github.com/paritytech/polkadot-sdk/pull/7402#discussion_r1940445322

Reply:
Is this comment on some outdated code? Currrently we use `UnpaidExecution`, so I don't think there is a need to convert location.

Reply:
For now, lets rather use [Convert](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_frame/traits/trait.Convert.html) than `TryConvert`. Its guaranteed on AH that account ids can be converted to Locations.

Reply:
Added: https://github.com/vgeddes/polkadot-sdk/pull/23/files#diff-bfe761b371a553619edf5be2dd792c45bd14f1a916e51b77f5b77b9395d92cdaR123 I couldn't find an existing impl for the Account to Location convert so I added one here: https://github.com/vgeddes/polkadot-sdk/pull/23/files#diff-de935282e0f68dc275742ff79668d81a33dc5b57edb776392e26d5f666e70437R109-R120